master/internal/api_user_intg_test.go

```diff
@@ -924,3 +926,207 @@ func getActivityEntry(ctx context.Context, userID model.UserID, entityID int32)
 	return db.Bun().NewSelect().Model((*model.UserActivity)(nil)).Where("user_id = ?",
 		int32(userID)).Where("entity_id = ?", entityID).Count(ctx)
 }
+
+func TestPostLongLivedToken(t *testing.T) {
+	api, curUser, ctx := setupAPITest(t, nil)
+
+	// Without lifespan input
```

Comment:
it might be worth checking to make sure the the lifespan is set to the default in this case as well, that way we can catch any regression in behavior if for some reason this endpoint breaks in the future when leveraging the default lifespan

Reply:
Done